### PR TITLE
V3 pipelines

### PR DESCRIPTION
**Pull request by sofiabravo103**
## Details

**Ticket URL**: Add materialized views as pipelines (many tickets)

**Description**: 

## TODO

Still something missing? You can set some checklists in the ticket.

- [x] Write unit tests.
- [ ] Write documentation.

## Checklist

Did I complete all the items in this checklist? Mark them when you did it.

- [ ] I self-reviewed my code. 
- [ ] I wrote the necessary tests.
- [ ] I wrote the necessary documentation (README, ADR...)
- [ ] I tested in my local dev machine that this code is running.
- [ ] I did lint my code (Python/YAML/SQL)
- [ ] I did bump the version *if needed*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This repo offers semi-automated materialized view creation. The sql files for vi
 - To live: From the **dwh-cluster-automation/plays** directory: `ansible-playbook -i ../live.ini ad-hoc/cdh-deployment.yml -t deploy-views-dags,restart-airflow`
 
 ### Create incremental table
-To automatically crate partitioned table out of incremental view query use `create_table.sh` script under `bimaterializedviewsautomation`.
+To automatically crate partitioned table out of incremental view query use `create_table.sh` script under `bimaterializedviewsautomation/manual_scripts`.
```
Comment (ElenaChi) [inline README.md]: Just a small typo: "create"

Comment (sofiabravo103) [reply README.md]: Thanks!

**Comment by ElenaChi** [comment]
+1

**Comment by KaivG** [comment]
+1

**Pull request merged**